rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
+
```

Comment:
- [ ] Getters and Setters for @book and @person instance variables should be present here as we are using this in our Rental class now.

Reply:
Thanks! Done